module/applications/advancement/advancement-manager.mjs

```diff
@@ -465,7 +473,7 @@ export default class AdvancementManager extends Application5e {
    * @returns {number}       Working level.
    */
   static currentLevel(item, actor) {
-    return item.system.levels ?? item.class?.system.levels ?? actor.system.details.level ?? 0;
+    return item.system.advancementLevel ?? actor.system.details.level ?? 0;
```

Comment:
`advancementLevel` is guaranteed to return a number, so using the nullish coalescing operator will have no effect. If the `item`'s `actor` here differs from the `actor` passed-in, we will also get the wrong result. I assume the intention is that we always read from the passed-in `actor` and ignore the `item's `actor`?

Reply:
This was more protection against cases where a module-provided item that doesn't inherit from `ItemDataModel` is used, in which case `advancementLevel` won't exist.

Reply:
I wonder why we pass-in the `actor` param then if it can always be read from the `item`?